frontend/public/components/routes.tsx

```diff
@@ -267,6 +222,22 @@ const RouteTargetRow = ({route, target}) => <tr>
   <td>{calcTrafficPercentage(target.weight, route)}</td>
 </tr>;
 
+const RouteIngressStatus: React.SFC<RouteIngressStatusProps> = ({ingresses}) =>
+  <React.Fragment>
+    {_.map(ingresses, (ingress) =>
+      <React.Fragment key={ingress.host}>
+        <h1 className="co-section-title">{ingress.host}</h1>
+        <dl>
+          <dt>Router</dt>
+          <dd>{ingress.routerName}</dd>
+          <dt>Wildcard Policy</dt>
+          <dd>{ingress.wildcardPolicy}</dd>
+        </dl>
+        <h4 className="co-m-route-ingress-conditions___title">Conditions</h4>
```

Comment:
Really this should be an `h2` semantically. I wonder if that adds more margin.

Not necessary to change in this PR